src/tests/test_openmmio.py

```diff
@@ -12,26 +12,28 @@
 """
 The testing functions for this class are located in test_target.py.
 """
+
 class TestLiquid_OpenMM(TargetTests):
     def setup_method(self, method):
-        pytest.skip("Needs optimizing to reduce runtime")
         super(TestLiquid_OpenMM, self).setup_method(method)
+        self.check_grad_fd = False
         # settings specific to this target
         self.options.update({
                 'jobtype': 'NEWTON',
                 'forcefield': ['dms.xml']})
 
         self.tgt_opt.update({'type':'LIQUID_OPENMM',
-            'name':'dms-liquid'})
+            'name':'dms-liquid', 'liquid_md_steps':500})
```

Comment:
(not blocking) I'm just curious - is this the change that enables the liquid tests?